Previous Events Images with a proper component

V-prajit opened the pull request:
Relates to #59 
Added the previous GBM image and created a proper component where you can simply input the event name and the image path. The page will scale accordingly, making it dynamic as well.

V-prajit committed:
Previous Events Images with a proper component

Added the previous GBM image and created a proper component where you can simply input the event name and the image path. The page will scale accordingly, making it dynamic as well.

atiqurx commented on src/components/events/EventCard.tsx:
```diff
@@ -0,0 +1,26 @@
+      className={`bg-white shadow-lg rounded-lg p-6 text-center border border-gray-200 flex flex-col items-center 
+      ${isSingle ? "w-full md:w-3/4 lg:w-2/3 xl:w-1/2 mx-auto" : "w-full"}`}
+    >
+      <h2 className="text-2xl font-semibold mb-4 text-gray-800">{eventName}</h2>
```
change the color to the text to white: text-white

atiqurx commented on src/components/events/EventCard.tsx:
```diff
@@ -0,0 +1,26 @@
+const EventCard: React.FC<EventCardProps> = ({ eventName, image1, image2, isSingle }) => {
+  return (
+    <div
+      className={`bg-white shadow-lg rounded-lg p-6 text-center border border-gray-200 flex flex-col items-center 
```
instead of using just using plain white bg, use the same bg we used in other cards. bg-white/10 backdrop-blur-md. change the border as well to border-white/40

atiqurx reviewed (commented):
Please do the following changes. Also add events from fall '24 if possible.

V-prajit committed:
Made the changes requested

Changes the text to white
made the background blurry
added the assets that are on the rest of the pages onto the events page as well
added the first town hall of fall 2024 as well

V-prajit commented:
Done
